src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySink.java

```diff
@@ -120,10 +118,9 @@ public void transform(StructuredRecord input, Emitter<KeyValue<Text, NullWritabl
         }
       }
       writer.endObject();
+      emitter.emit(new KeyValue<>(writer.get().getAsJsonObject(), NullWritable.get()));
```

Comment:
Just curious, how come the sink works regardless of whether you send a JsonObject or Text? Is the underlying output format handling both?

Reply:
yes, underlying TextOutputFormat handles `JsonObject` and `Text` objects.
![image](https://user-images.githubusercontent.com/14131070/59939600-50a53200-940d-11e9-81d2-eeda50a7316d.png)